server/channels/app/export.go

```diff
@@ -902,7 +907,11 @@ func (a *App) copyEmojiImages(emojiId string, emojiImagePath string, pathToDir s
 	if fromPath == nil || err != nil {
 		return errors.New("Error reading " + emojiImagePath + "file")
 	}
-	defer fromPath.Close()
+	defer func() {
+		if err = fromPath.Close(); err != nil {
+			a.Log().Error("Error closing file", mlog.Err(err))
```

Comment:
Can we add a `request.CTX` to `copyEmojiImages` and use it's `Logger()` instead of the global one? Thanks!